cmd/juju/action/showoutput.go

```diff
@@ -153,8 +153,12 @@ func timerLoop(api APIClient, requestedId string, wait, tick *time.Timer) (param
 		// Block until a tick happens, or the timeout arrives.
 		select {
 		case _ = <-wait.C:
-			return result, nil
-
+			switch result.Status {
+			case params.ActionRunning, params.ActionPending:
+				return result, errors.NewTimeout(err, "deadline reached")
```

Comment:
"timeout reached" perhaps?

Reply:
Wondered about that myself.. I'll make the change.